### PR TITLE
Update release note with OpenAPI changes 

### PR DESCRIPTION
**Pull request by lnash94**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md [hunk 1]
```diff
@@ -231,7 +231,22 @@ Introduced the `--native` flag, which generates a GraalVM native executable when
...
+- Introduced `@openapi:ClientConfig` annotation to add additional details to create IDL client.
+```ballerina
+@openapi:ClientConfig {
+    tags: ["store"],
+    operations: ["op1", "op2"],
+    license: "/path/to/license_file.txt",
+    isResource: true // (default value => true),
+    nullable: false // (default value => false)
+}
+client "./openapi.yaml" as foo;
```
Comment (NipunaRanasinghe) [inline release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md:248]:
```suggestion
@openapi:ClientConfig {
    tags: ["store"],
    operations: ["op1", "op2"],
    license: "/path/to/license_file.txt",
    isResource: true, // default value => true
    nullable: false // default value => false
}
client "./openapi.yaml" as foo;
```

File: release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md [hunk 2]
```diff
@@ -231,7 +231,33 @@ Introduced the `--native` flag, which generates a GraalVM native executable when
...
 - `bal build --native`
 
+#### OpenAPI Tool
+- Introduced non-Ballerina IDL client declaration support for openAPI specifications. With the support, openAPI specifications can be imported(i.e. declared) at module level and, consumed as follows.
+```ballerina
+// Add the client import with OpenAPI specification.
+client "./openapi.yaml" as weather; 
+
+public function main() returns error? {
+    // Initialize the client , which is generated by IDL import.
```
Comment (NipunaRanasinghe) [inline release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md:241]:
```suggestion
    // Initializes the client which is generated from the IDL import.
```

File: release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md [hunk 3]
```diff
@@ -231,7 +231,33 @@ Introduced the `--native` flag, which generates a GraalVM native executable when
...
+    weather:client weatherClient = check new ();
+    // Consume the `/weather` API, that has been given in the OpenAPI specification.
```
Comment (NipunaRanasinghe) [inline release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md:243]:
```suggestion
    // Consume the `/weather` resource defined in the OpenAPI specification.
```

File: release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md [hunk 4]
```diff
@@ -231,7 +231,33 @@ Introduced the `--native` flag, which generates a GraalVM native executable when
...
+    weather:CurrentWeatherData weatherResult = check weatherClient->/weather("Colombo");
+}
+```
+- Also introduced `@openapi:ClientConfig` annotation to add additional details to create IDL client.
```
Comment (NipunaRanasinghe) [inline release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md:247]:
```suggestion
Also introduced `@openapi:ClientConfig` annotation to configure the client generation.
```

**Review by NipunaRanasinghe — APPROVED** [review]
LGTM

File: release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md [hunk 5]
```diff
@@ -231,7 +231,33 @@ Introduced the `--native` flag, which generates a GraalVM native executable when
 - The generated executable contains the modules in the current package, their dependencies, Ballerina runtime, and statically linked native code from the JDK.
 - `bal build --native`
 
+#### OpenAPI Tool
+- Introduced non-Ballerina IDL client declaration support for openAPI specifications. With the support, openAPI specifications can be imported(i.e. declared) at module level and, consumed as follows.
```
Comment (praneesha) [inline release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md:235]:
```suggestion
- Introduced non-Ballerina IDL client declaration support to import OpenAPI specifications to declare and consume them at the module level as follows.
```

File: release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md [hunk 6]
```diff
@@ -231,7 +231,33 @@ Introduced the `--native` flag, which generates a GraalVM native executable when
...
+```ballerina
+// Add the client import with OpenAPI specification.
```
Comment (praneesha) [inline release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md:237]:
```suggestion
// Add the client import with the OpenAPI specification.
```

File: release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md [hunk 7]
```diff
@@ -231,7 +231,33 @@ Introduced the `--native` flag, which generates a GraalVM native executable when
...
+client "./openapi.yaml" as weather; 
+
+public function main() returns error? {
+    // Initializes the client which is generated from the IDL import.
+    weather:client weatherClient = check new ();
+    // Consume the `/weather` resource defined in the OpenAPI specification.
+    weather:CurrentWeatherData weatherResult = check weatherClient->/weather("Colombo");
+}
+```
+Also introduced `@openapi:ClientConfig` annotation to configure the client generation.
```
Comment (praneesha) [inline release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md:247]:
```suggestion
Also, introduced the `@openapi:ClientConfig` annotation to configure the client generation.
```

File: release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md (hunk 5 above)
Comment (praneesha) [reply release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md:235]: @NipunaRanasinghe - Please check the changes.

Comment (NipunaRanasinghe) [reply release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md:235]: This seems incorrect as the client declaration is only getting declared (but not getting consumed) at the module level. Shall we modify accordingly?

Comment (praneesha) [reply release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md:259]: @lnash94 - Please change as follows:

- Introduced non-Ballerina IDL client declaration support for OpenAPI specifications. With this support, OpenAPI specifications can now be imported (i.e., declared) at the module level and consumed as follows.

**Review by NipunaRanasinghe — DISMISSED** [review]
LGTM

**Review by praneesha — APPROVED** [review]

**Pull request merged**